documentation/content/codingstyle.rst

```diff
@@ -180,6 +180,13 @@ $PATH, you should use 'type ', instead of 'which '.
 The output of 'which' is not machine parsable and its exit code
 is not reliable across platforms.
 
+Sourcing files
```

Comment:
Maybe something like "How to include code" or something similar that gives the idea of inclusion operation.